sources/install/package_base.sh

```diff
@@ -391,18 +388,18 @@ function package_base() {
     cp -v /root/sources/assets/bash/bashrc ~/.bashrc
 
     # setup Python environment
+    # the order matters (if 2 is before 3, `python` will point to Python 2)
+    PYTHON_VERSIONS="3 3.11 3.6 2"
     install_pyenv
     pip2 install --no-cache-dir virtualenv
     # https://stackoverflow.com/questions/75608323/how-do-i-solve-error-externally-managed-environment-everytime-i-use-pip3
     # TODO: do we really want to unset EXTERNALLY-MANAGED? Not sure it's the best course of action
     # with pyenv, not sure the command below is needed anymore
     # rm /usr/lib/python3.*/EXTERNALLY-MANAGED
-    pip3 install --upgrade pip
-    pip3.6 install --upgrade pip
-    pip2 install --upgrade pip
-    pip3 install wheel
-    pip3.6 install wheel
-    pip2 install wheel
+    for v in $PYTHON_VERSIONS; do
+        pip${v} install --upgrade pip
+        pip${v} install wheel
+    done
     install_pipx
```

Comment:
I think pipx will be installed with the latest python3 here. Which will probably prevent `pipx --python python3.11` from working, what do you think? Have you tested?

Reply:
Yes, I tested that and it found `python3.11` in the pyenv installation.

Reply:
not applicable anymore since python3.12 is out of the picture for now